src/public/index.html

```diff
@@ -9,37 +9,54 @@
   </head>
 
 
-    <body>
-      <header>
-        <h1>Something Awesome!</h1>
-      </header>
-      <nav>
-        <ul>
-          <li><a href="/about/">About</a></li>
-          <li> <a href="/contact/">Contact</a></li>
-        </ul>
-      </nav>
+  <body>
+    <header>
+      <div class="title">
+        <h1 class="plaintext-color">
+            Something Awesome! <br/>
+        </h1>
+      </div>
+    </header>
+      <div class="content-container page">
```

Comment:
This `<div>` opening tag should be at the same level of indentation as the closing `</header>` tag above it. Most IDEs have an auto-format feature, or you can even use JSFiddle's "Tidy" button to fix indentation.